winsup/utils/getprocaddr.c

```diff
@@ -134,11 +135,11 @@ ctrl_handler (DWORD ctrl_type)
           return 1;
         }
       /* Inject the remote thread only when asked to */
-      int t = inject_remote_thread_into_process (h, address, exit_code);
+      int t = inject_remote_thread_into_process (h, address, exit_code, &thread_return);
```

Comment:
Do you need this variable `t`? It will be either `0` or `-1` so rather than assigning to `t` it can be directly checked in an `if` condition.

Also, I would like `thread_return` to be printed.